cli/sonica-cli.py

```diff
@@ -33,8 +35,8 @@ def skip():
         print(ansi(31, r.reason))
 
 @cli.command()
-def search(query : str, engines : str = ''):
-    q = [query] # The daemon asks for a list of queries
+def search(query : List[str], engines : str = ''):
```

Comment:
I fail to see the purpose of parsing the query like this, instead of just doing `[str]`

Reply:
Nevermind, I read this completely wrong :D